lib/main.coffee

```diff
@@ -10,70 +10,52 @@ module.exports =
       default: 'foodcritic'
     extraArgs:
       type: 'string'
-      default: ''
 
   activate: ->
     @subscriptions = new CompositeDisposable
     @MessageRegexp = null
-    @subscriptions.add atom.config.observe 'foodcritic.executablePath', =>
-      @executablePath = atom.config.get 'linter-foodcritic.executablePath'
-    @subscriptions.add atom.config.observe 'foodcritic.extraArgs', =>
-      @extraArgs = atom.config.get 'linter-foodcritic.extraArgs'
+    @subscriptions.add atom.config.observe 'foodcritic.executablePath', (executablePath) =>
```

Comment:
`'foodcritic.` this should be `'linter-foodcritic.`, aka the package name in manifest